underactuated.html

```diff
@@ -464,12 +467,13 @@ <h1>Honda's ASIMO vs. passive dynamic walkers</h1>
       announced that they had been working for nearly 15 years (behind closed
       doors) on walking robot technology.  Their designs have continued to
       evolve, resulting in a humanoid robot they call ASIMO (Advanced Step in
-      Innovative MObility).  Honda's ASIMO is widely considered to be state of
-      the art in walking robots, although there are now many robots with designs
-      and performance very similar to ASIMO's. We will dedicate effort to
-      understanding a few of the details of ASIMO when we discuss algorithms for
-      walking... for now I just want you to become familiar with the look and
-      feel of ASIMO's movements [watch the asimo video below now]. </p>
+      Innovative MObility).  For nearly 20 years, Honda's robots were widely
```

Comment:
agree I think this works well!

and sets up nicely for the Atlas comparison at end of the Intro